README.md

```diff
@@ -33,14 +33,23 @@ To refresh auto-generated code, use the `generate` command. Usually, plugins try
 cfn generate
 ```
 
+### Command: submit
+
+To submit resource providers package for registration, use the `submit` command. This will prepare necessary artifacts to register a resource AWS account.
```

Comment:
```suggestion
To register a resource provider, use the `submit` command. This will prepare necessary artifacts to register a resource in your AWS account.
```